container/bin/nbd-server.py

```diff
@@ -0,0 +1,41 @@
+#!/venv/bin/python3
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import argparse
+
+from ovirt_imageio.client import info
+from ovirt_imageio._internal import nbd
+from ovirt_imageio._internal import qemu_nbd
+
+
+DEFAULT_SOCKET_PATH = "/tmp/nbd.sock"
```

Comment:
This is fine in a container, but unsafe outside of a container. The right way
is to create a temporary directory using mkdtemp() with a ranfom name and strict
permissions, so nobody can attack you using a symlink in /tmp/nbd.sock.

See how we do this in client._api. The best way to reuse that code is likely to move it to _internal and use it from both client and server scrtips.

Reply:
I didn't think to use it outside the container for now. But as we provide it, it may be tempting to use it. So I'll try to make it safer with your suggestion.